dags/common/enricher.py

```diff
@@ -0,0 +1,106 @@
+import xml.etree.ElementTree as ET
+from typing import Dict
+
+import backoff
+import requests
+from structlog import get_logger
+
+
+class Enricher(object):
+    def __init__(self) -> None:
+        self.logger = get_logger().bind(class_name=type(self).__name__)
+
+    def _get_schema(self):
+        return "http://repo.scoap3.org/schemas/hep.json"
```

Comment:
this should not be hardcoded, we will run the code in different environments hence we will have different schema paths i.e. http://repo.qa.scoap3.org

Reply:
I didn't know about that. Done 👍